mantis-control-plane/mantis-control-plane-core/src/test/java/io/mantisrx/server/master/resourcecluster/TestTaskExecutorRegistration.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.mantisrx.server.master.resourcecluster;
+
+import static org.junit.Assert.assertEquals;
+
+import io.mantisrx.common.JsonSerializer;
+import org.junit.Test;
+
+public class TestTaskExecutorRegistration {
```

Comment:
nit: we should follow convention. call this `TaskExecutorRegistrationTest`. Move the heartbeat test to `TaskExecutorHeartbeatTest`.

Reply:
Renamed the class.